src/components/implementation/cbuf_mgr/default/cbuf_mgr.c

```diff
@@ -62,8 +72,8 @@ struct cbuf_maps {
 };
 
 struct cbuf_info {
-	u32_t cbid;
-	int size;
+	int cbid;
```

Comment:
I have a solution to keep this unsigned that involves using a separate unsigned variable in cbuf_create and some casting. Yuxin would like to minimize casting (I assume for code cleanliness) so we are not sure. I am fine with the code as-is just to close out the merge. Then we can look at my solution a bit more and see if we are ok with casting. The input parameter to cbuf_create and the return value have to be ints so some casting will be needed.